logger.go

```diff
@@ -690,6 +690,7 @@ func (e *Entry) Time(key string, t time.Time) *Entry {
 	if e == nil {
 		return nil
 	}
+
```

Comment:
While this change (and the other ones new line changes) is good for readability, I would have put them in a dedicated commit (or PR)

It would be easier to review